contrib-zypper/README.md

```diff
@@ -0,0 +1,35 @@
+# Zypper
+
+Defines [zypper][1] aliases.
+
+## Aliases
+
+* `zyp` # Zypper.
+* `zypc` # Cleans the cache.
+* `zypi` # Installs package(s).
+* `zypl` # Clean packages cache.
+* `zypq` # Displays package information.
+* 'zypre' # Refreshes repositories.
```

Comment:
Looks like the wrong quotes on this line

Reply:
sorry I haven't checked errors!!!